content/source/docs/enterprise/install/uninstall.html.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: "enterprise"
+page_title: "Uninstall - Terraform Enterprise"
+---
+
+# Uninstall Terraform Enterprise
+
+If you installed Terraform Enterprise on VMWare instances, you may not be able to easily request new virtual machines for a broken or corrupted installation. Instead, you can use the `uninstall` script to remove Terraform Enterprise and all of its services (excluding Docker) from a system.
+
+~> **Important**: This script does not touch the mounted disk path, so you will need to manually clean that up if necessary.
+
+Please contact [support][support] with questions or issues.
+
+## While the Script Runs
+
+After you initiate `uninstall`:
+
+1. Enter `yes` when asked if you want to continue with the installation.
+
+	```
+	$ sudo ./uninstall.sh 
+	This script will completely uninstall Terraform Enterprise and Replicated on this system.
+	Do you wish to continue? (y/n)yes
+	Proceeding with uninstall...
+	Stopping and disabling the replicated services...
+	Removed /etc/systemd/system/docker.service.wants/replicated-operator.service.
+	Removed /etc/systemd/system/docker.service.wants/replicated.service.
+	Removed /etc/systemd/system/docker.service.wants/replicated-ui.service.
+	Replicated services stopped and disabled.
+	Stopping any running application containers
+	a8dd38ebcc67
+	11a3c7d476a3
+	45f786c7c632
+	31664ff29012
+	c0122c1dce13
+	8c6b98b6d498
+	d3856e83038d
+	b5e160999e2b
+	a7f69f5507de
+	45a07127fa15
+	7a364889673d
+	d5f6218e1d3c
+	3f10e192a503
+	818bb2464291
+	1bd96c43b48d
+	8388555131dd
+	0a0cb265a2bf
+	rabbitmq
+	telegraf
+	influxdb
+	anchor_isolation_network
+	Removing Replicated Docker containers...
+	replicated
+	replicated-ui
+	replicated-operator
+	replicated-premkit
+	replicated-statsd
+	retraced-api
+	retraced-processor
+	retraced-cron
+	retraced-nsqd
+	retraced-postgres
+	Removing Replicated files and executables...
+	Run systemctl daemon-reload...
+	Terraform Enterprise and Replicated should now be uninstalled.
+	```
+
+   The script stops the Replicated services, removes the Docker containers, and removes Replicated executables and configuration files from the system.  
+
+
+2. Choose an option to prune Docker volumes:  
```

Comment:
This is showing up as oddly indented when I test locally - is there any way to fix it?

Reply:
I think this is fixed now.